frontend/src/tests/application-tests.js

```diff
@@ -1,19 +1,110 @@
-import { it, beforeAll } from "./utils";
+import {
+    it,
+    beforeAll,
+    expect,
+    checkPropTypes,
+    postingPropTypes,
+} from "./utils";
 import { databaseSeeder } from "./setup";
 
 export function applicationsTests({ apiGET, apiPOST }) {
-    beforeAll(async () => {
-        await databaseSeeder.seed({ apiGET, apiPOST });
-    }, 30000);
+    let session;
+    let position;
+    let posting = {};
+    let resp;
+    let admin;
+    let surveyData;
+
+    const taOnlyUser = {
+        utorid: "matthewc",
+        roles: ["ta"],
+    };
+
+    const postingData = {
```

Comment:
Do we expect this to be reused in other tests too?

Reply:
The seeder's to come in a different PR as I can't combine two issues in one PR.
After submitting postingData, we store as a global.